install/centos-installation-guide.md

```diff
@@ -23,7 +23,7 @@
    ```bash
    $ source /etc/os-release
    $ sudo yum -y install yum-utils
-   $ sudo -E VERSION_ID=$VERSION_ID yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/release/CentOS_${VERSION_ID}/home:katacontainers:release.repo"
+   $ sudo -E VERSION_ID=$VERSION_ID yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/master/CentOS_${VERSION_ID}/home:katacontainers:releases:x86_64:master.repo"
```

Comment:
Our CI will actually execute this code since the following conditions are true:

- it's an install guide
- it's a `bash` code block.

Since we plan to offer OBS packages for other architectures "real soon now", please could you add a variable for the architecture. Something like:

```
$ arch=$(arch)
$ sudo -E VERSION_ID=$VERSION_ID yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${arch}:/master/CentOS_${VERSION_ID}/home:katacontainers:releases:${arch}:master.repo"
```

That way, we won't need to update these docs when we *do* support other architectures in OBS ;)

Same comment for the other install guides.

Reply:
I am fine with it as a short term solution.
In the long term a single OBS project should be capable to deliver packages for different architectures.

Reply:
I should probably remove the `(currently x86_64 only)` from the guides too!

Reply:
Agreed - there must be a way for yum/apt to send their client arch to the server and have OBS redirect transparently to the right project.